lib/Sema/TypeCheckError.cpp

```diff
@@ -134,6 +134,9 @@ class AbstractFunction {
         fn = conversion->getSubExpr()->getValueProvidingExpr();
       } else if (auto conversion = dyn_cast<BindOptionalExpr>(fn)) {
         fn = conversion->getSubExpr()->getValueProvidingExpr();
+      // Look through optional injections
```

Comment:
For consistency, this should end in a period.

Reply:
Indeed. I'll tack that on in some other random PR I land today